lib/plug/adapters/cowboy/conn.ex

```diff
@@ -31,6 +31,13 @@ defmodule Plug.Adapters.Cowboy.Conn do
   end
 
   def send_resp(req, status, headers, body) do
+    statuses = Application.get_env(:plug, :statuses, %{})
```

Comment:
I think we should move this to compile time, however it makes the testing a little trickier. What is the recommended way to recompile the module in the tests?